dist/docker/redhat/build_docker.sh

```diff
@@ -15,19 +15,12 @@ version="$(sed 's/-/~/' <build/SCYLLA-VERSION-FILE)"
 release="$(<build/SCYLLA-RELEASE-FILE)"
 
 mode="release"
-type="ubuntu"
 
-if uname -m | grep x86_64 ; then
-  arch="amd64"
-fi
-
-if uname -m | grep aarch64 ; then
-  arch="arm64"
-fi
+arch="$(uname -m)"
 
 
 print_usage() {
-    echo "usage: $0 [--mode mode] [--type ubuntu|pro, default: ubuntu]"
+    echo "usage: $0 [--mode mode]"
```

Comment:
Need to check it runs on a FIPS enabled host!
I assume UBI9 is FIPS enabled?
https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/security_hardening/switching-rhel-to-fips-mode_security-hardening#enabling-fips-mode-in-a-container_using-the-system-wide-cryptographic-policies

Reply:
@syuu1228 note that change required also change in pkg

And they need to be merged together

I will prepare one based on your branch

Reply:
> Unlike Ubuntu, there is no "pro" image, default image supported FIPS mode.

Thanks @syuu1228 - this is what I had hoped will be the case!